cmd/run.go

```diff
@@ -176,13 +178,24 @@ var RunCmd = &cobra.Command{
 			err = output.AppCMD.Start()
 			if err != nil {
 				print.FailureStatusEvent(os.Stdout, err.Error())
-				os.Exit(1)
+				appRunning <- false
+				return
 			}
 
 			appRunning <- true
 		}()
 
-		<-appRunning
+		appRunStatus := <-appRunning
+		if !appRunStatus {
+			// Start App failed, try to stop Dapr and exit.
+			err = output.DaprCMD.Process.Kill()
```

Comment:
@wcs1only Do you think this is sufficient in windows for the process to be killed?

Reply:
Yeah, should be fine. In general, the CLI can kill a process it started. The issue we fixed a few weeks ago was related to running `dapr stop` from another terminal (killing a process that isn't your child is not something you can do gracefully on Windows).